DXRMapFixups/DeusEx/Classes/DXRFixupParis.uc

```diff
@@ -394,8 +394,27 @@ function AnyEntryMapFixes()
 function PostFirstEntryMapFixes()
 {
     local #var(prefix)WIB wib;
+    local #var(prefix)NicoletteDuclare nico;
 
     switch(dxr.localURL) {
+    case "10_PARIS_METRO":
+        if (dxr.flags.settings.starting_map >= 109) {
+            foreach AllActors(class'#var(prefix)NicoletteDuclare', nico, 'DXRMissions') {
+                nico.LeaveWorld();
+                break;
+            }
+        }
+        break;
+
+    case "10_PARIS_CLUB":
+        if (dxr.flags.settings.starting_map >= 109) {
+            foreach AllActors(class'#var(prefix)NicoletteDuclare', nico, 'DXRMissions') {
```

Comment:
vanilla code actually handles this already in Mission10.uc FirstFrame()

```
	else if (localURL == "10_PARIS_CLUB")
	{
		if (flags.GetBool('NicoletteLeftClub'))
		{
			foreach AllActors(class'NicoletteDuClare', Nicolette)
				Nicolette.Destroy();
		}
	}
```
```suggestion
```

Reply:
You're right, it was unnecessary.